src/PaymentPart/Output/FpdfOutput/FpdfOutput.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput;
+
+use Sprain\SwissQrBill\PaymentPart\Output\AbstractOutput;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\OutputElementInterface;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Text;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Title;
+use Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput\Template\QrBillFooter;
+use Sprain\SwissQrBill\PaymentPart\Output\OutputInterface;
+use Sprain\SwissQrBill\PaymentPart\Translation\Translation;
+use Sprain\SwissQrBill\QrBill;
+use Sprain\SwissQrBill\QrCode\QrCode;
+
+final class FpdfOutput extends AbstractOutput implements OutputInterface
+{
+    /** @var QrBillFooter */
+    private $fpdf;
+
+    /** @var float */
+    private $amountLS = 0;
+
+    public function __construct(
+        QrBill $qrBill,
+        string $language,
+        QrBillFooter $fpdf
+    ) {
+        parent::__construct($qrBill, $language);
+        $this->fpdf = $fpdf;
+        $this->setQrCodeImageFormat(QrCode::FILE_FORMAT_PNG);
```

Comment:
`setQrCodeImageFormat()` should be re-defined in this class and throw an exception when something wants to set the image format to svg, as this is not supported by Fpdf.

Reply:
I will try to do that...

Reply:
Thanks for your work here! I appreciate your help.